ui/src/domain/jsonSchema.ts

```diff
@@ -45,9 +45,30 @@ export type NumberAttribute = {
   type: "number";
 };
 
+type StringFormat =
```

Comment:
Why do we need it if below we accept any string?

Reply:
I started creating this as a comment and then moved it to a type but yes, this could be a comment. It's here mainly for documentation purposes so that we know what the standard types are.

Reply:
I would add this to the UI README, like we did with the JSON Schema support.

Reply:
Yep, I like that! I've also specified that:

> only `date-time`, `time` and `date` will show specialized inputs